pandas/tests/extension/base/getitem.py

```diff
@@ -82,8 +82,9 @@ def test_getitem_scalar(self, data):
         assert isinstance(result, data.dtype.type)
 
     def test_getitem_scalar_na(self, data_missing, na_cmp, na_value):
-        result = data_missing[0]
-        assert na_cmp(result, na_value)
+        if data_missing._can_hold_na:
```

Comment:
All these changes are a bit unfortunate...

Could we instead have the `data_missing` fixture raise `pytest.skip` when necessary? I suspect this would have to be done by the EA author in their code.

Reply:
I could make that change to avoid any tests where `data_missing` is used, and the EA author would have to make that change if they wanted `_can_hold_na=False`.